spec/index.html

```diff
@@ -1802,42 +1814,59 @@ <h3>RDF Triples Constructors</h3>
         |curPredicate| <var>N</var> .</span>
     </p>
 
-    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms:</span></h4>
+    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms</span></h4>
 
     <p>Beginning the <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> production
       records the |curSubject| and |curPredicate|.
       Finishing the <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> production
-      and restores the recorded values of |curSubject| and |curPredicate|.</p>
+      restores the recorded values of |curSubject| and |curPredicate|.</p>
+
+    <h4 id="reifier" style="padding-bottom:0; margin-bottom:0;"><span>Reifier</span></h4>
+
+    <p>Beginning the <a href="#grammar-production-reifier"><code>reifier</code></a> production,
+      if |curReifier| is set, the production yields the RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|.
+      Finishing the <a href="#grammar-production-reifier"><code>reifier</code></a> production,
+      |curReifier| is taken from the <a href="#handle-reifier"><code>reifier</code></a> term constructor.</p>
 
-    <h4 id="reifier" style="padding-bottom:0; margin-bottom:0;"><span>Reifiers:</span></h4>
+    <h4 id="reifiedTriple" style="padding-bottom:0; margin-bottom:0;"><span>Reified Triple</span></h4>
 
     <p>Beginning the <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       records the |curSubject| and |curPredicate|.
       |curSubject| is taken from the
-      <a href="#handle-reifiedTriple"><code>reifiedTriple</code></a> term constructor.
+      <a href="#grammar-production-reifier"><code>reifier</code></a> of the
+      <a href="#handle-reifiedTriple"><code>reifiedTriple</code></a> term constructor
+      if present,
+      or a fresh RDF blank node, if not matched.
       A new <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> instance |TT|
       is composed from
       the <a href="#grammar-production-subject"><code>subject</code></a>,
       <a href="#grammar-production-predicate"><code>predicate</code></a>, and
       <a href="#grammar-production-object"><code>object</code></a> productions.
-      Finishing the <a href="#grammar-production-reifier"><code>reifier</code></a> production
+      Finishing the <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       yields the RDF triple |curSubject| <code>rdf:reifies</code> |TT|
```

Comment:
`curSubject` may have been modified by the subject/reifiedTriple in the subject position insiude `<< >>`.
It might be batter use `curReifier` and make it a stack (for annotation and asserted triples it does not need to be a stack as in comment above.